src/main/MainScreen.js

```diff
@@ -157,8 +158,9 @@ class MainScreen extends React.Component {
 
 const mapStateToProps = (state) => ({
   auth: getAuth(state),
-  subscriptions: state.subscriptions,
+  subscriptions: state.realm.subscriptions,
   messages: state.stream.messages,
+  twentyFourHourTime: state.realm.twenty_four_hour_time,
```

Comment:
We strive to have flat Redux stores.
That means we want a subscriptions store, and the meta data can be stored in another store, maybe called realm.

So instead of:

```
{
  subscriptions: {
    realm: {
      subscriptions,   
      twentyFourHourTime,
    }
  }
}
```

We want:

```
{
  subscriptions,
  realm: {
    twentyFourHourTime,
  },
}
```